addons/html_editor/static/src/main/media/image_plugin.js

```diff
@@ -277,27 +277,27 @@ export class ImagePlugin extends Plugin {
         }
     }
 
-    getSelectedImage() {
```

Comment:
Same. Just to be clear, we just need to keep the function entry point, but it can execute the code of the new one.